app/src/main/java/com/poupa/vinylmusicplayer/adapter/GenreAdapter.java

```diff
@@ -87,7 +87,7 @@ public int getItemCount() {
     @Override
     public String getSectionName(int position) {
         final Genre genre = dataSet.get(position);
-        return genre.id == -1 ? "" : MusicUtil.getSectionName(dataSet.get(position).name);
+        return genre.id == -1 ? "" : MusicUtil.getSectionName(dataSet.get(position).getName());
```

Comment:
shouldn't we return the default genre instead of the empty string here?

Reply:
make sense